src/test/integrasjonstester/kotlin/no/nav/familie/ba/sak/kjerne/beregning/SatsServiceIntegrationTest.kt

```diff
@@ -1,95 +1,73 @@
 package no.nav.familie.ba.sak.kjerne.beregning
 
 import no.nav.familie.ba.sak.config.AbstractSpringIntegrationTest
-import no.nav.familie.ba.sak.kjerne.beregning.domene.SatsType
-import org.junit.jupiter.api.Assertions
+import no.nav.familie.ba.sak.kjerne.beregning.domene.SatsType.ORBA
+import no.nav.familie.ba.sak.kjerne.beregning.domene.SatsType.SMA
+import no.nav.familie.ba.sak.kjerne.beregning.domene.SatsType.TILLEGG_ORBA
+import no.nav.familie.ba.sak.kjerne.beregning.domene.SatsType.UTVIDET_BARNETRYGD
+import no.nav.familie.ba.sak.kjerne.eøs.util.tilTidslinje
+import no.nav.familie.ba.sak.kjerne.tidslinje.eksperimentelt.ogSenere
+import no.nav.familie.ba.sak.kjerne.tidslinje.eksperimentelt.ogTidligere
+import no.nav.familie.ba.sak.kjerne.tidslinje.eksperimentelt.plus
+import no.nav.familie.ba.sak.kjerne.tidslinje.tidsrom.rangeTo
+import no.nav.familie.ba.sak.kjerne.tidslinje.util.aug
+import no.nav.familie.ba.sak.kjerne.tidslinje.util.des
+import no.nav.familie.ba.sak.kjerne.tidslinje.util.feb
+import no.nav.familie.ba.sak.kjerne.tidslinje.util.jan
+import no.nav.familie.ba.sak.kjerne.tidslinje.util.mar
+import no.nav.familie.ba.sak.kjerne.tidslinje.util.sep
+import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
-import java.time.LocalDate
-import java.time.YearMonth
 
 class SatsServiceIntegrationTest : AbstractSpringIntegrationTest() {
```

Comment:
Trenger dette å være en integrasjonstest?
```suggestion
class SatsServiceIntegrationTest {
```

Reply:
Nei, det trengs ikke. Burde lage en rydde-oppgave....